data/autoyast/autoyast_btrfs.xml

```diff
@@ -166,4 +171,26 @@
       <username>root</username>
     </user>
   </users>
+  <report>
+    <errors>
+      <log config:type="boolean">true</log>

```

Comment:
As spoken, we need to find out if <log...> and <show...> are really needed here

Reply:
These are indeed default values, so removed from given profiles. Thanks!